crates/router/src/connector/bambora/transformers.rs

```diff
@@ -139,7 +139,21 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for BamboraPaymentsRequest {
                     term_url: item.request.complete_authorize_url.clone(),
                 })
             }
-            _ => Err(errors::ConnectorError::NotImplemented("Payment methods".to_string()).into()),
+            domain::PaymentMethodData::CardRedirect(_)
+            | domain::PaymentMethodData::Wallet(_)
+            | domain::PaymentMethodData::PayLater(_)
+            | domain::PaymentMethodData::BankRedirect(_)
+            | domain::PaymentMethodData::BankDebit(_)
+            | domain::PaymentMethodData::BankTransfer(_)
+            | domain::PaymentMethodData::Crypto(_)
+            | domain::PaymentMethodData::MandatePayment
```

Comment:
/MandatePayment and Reward missing (_)?

Reply:
I did not include the `(_)` syntax for those two because it causes an error when running `cargo check`.

To my understanding, the `(_)` syntax is only required when the enum variant holds additional data. The MandatePayment and Reward variants (defined [here](https://github.com/juspay/hyperswitch/blob/main/crates/router/src/types/domain/payments.rs#L8) are unit-only variants that do not include any extra data.